packages/wonder-blocks-core/src/text.js

```diff
@@ -44,10 +42,12 @@ export default class Text extends Component {
             this.props.style,
         ]);
 
+        const Tag = this.props.tag;
+
         return (
-            <this.props.tag style={style} className={className}>
+            <Tag style={style} className={className}>
```

Comment:
This change is a bit surprising - what was Flow's complaint here?

Reply:
I was surprised too.  I'll revert and post the error message tomorrow.

Reply:
```
/Users/kevinbarabash/khan/wonder-blocks/packages/wonder-blocks-core/src/text.js:48
  46 | 
  47 |         return (
> 48 |             <this.props.tag style={style} className={className}>
     |              ^^^^ this
  49 |                 {this.props.children}
  50 |             </this.props.tag>
  51 |         );

type referenced from value position

/Users/kevinbarabash/khan/wonder-blocks/packages/wonder-blocks-core/src/text.js:32
  30 | });
  31 | 
> 32 | export default class Text extends React.Component<Props> {
     |                      ^^^^ type this
  33 |     static defaultProps = {
  34 |         tag: "span",
  35 |     };
```
I'm not really sure what it means though.

Reply:
Huh! That's inscrutable...